<!DOCTYPE html>
<html>

  <head>
  <meta charset="utf-8">
  <meta http-equiv="X-UA-Compatible" content="IE=edge">
  <meta name="viewport" content="width=device-width, initial-scale=1">

  <title>Publications</title>
  <meta name="description" content="Institute of Applied Biosciences (INAB), Center for Research and Technology Hellas (CERTH), Thermi 57001, Greece
">

  <link rel="stylesheet" href="/css/main.css">
  <link rel="canonical" href="http://fpsom.github.io/research/publications/">
  <link rel="alternate" type="application/rss+xml" title="Cloud Computing and Bioinformatics" href="http://fpsom.github.io/feed.xml" />
  <link rel='stylesheet' id='open-sans-css'  href='//fonts.googleapis.com/css?family=Open+Sans%3A300italic%2C400italic%2C600italic%2C300%2C400%2C600&#038;subset=latin%2Clatin-ext&#038;ver=4.2.4' type='text/css' media='all' />
  <link href='https://fonts.googleapis.com/css?family=Titillium+Web:600italic,600,400,400italic' rel='stylesheet' type='text/css'>
</head>


  <body>

    <header class="site-header">

  <div class="wrapper">

    <a class="site-title" href="/">Cloud Computing and Bioinformatics</a>


    <nav class="site-nav">

      <a href="#" class="menu-icon menu.open">
        <svg viewBox="0 0 18 15">
          <path fill="#424242" d="M18,1.484c0,0.82-0.665,1.484-1.484,1.484H1.484C0.665,2.969,0,2.304,0,1.484l0,0C0,0.665,0.665,0,1.484,0 h15.031C17.335,0,18,0.665,18,1.484L18,1.484z"/>
          <path fill="#424242" d="M18,7.516C18,8.335,17.335,9,16.516,9H1.484C0.665,9,0,8.335,0,7.516l0,0c0-0.82,0.665-1.484,1.484-1.484 h15.031C17.335,6.031,18,6.696,18,7.516L18,7.516z"/>
          <path fill="#424242" d="M18,13.516C18,14.335,17.335,15,16.516,15H1.484C0.665,15,0,14.335,0,13.516l0,0 c0-0.82,0.665-1.484,1.484-1.484h15.031C17.335,12.031,18,12.696,18,13.516L18,13.516z"/>
        </svg>
      </a>  

    <div class="trigger"><h1>Main Navigation</h1>

 <ul class="menu">

    
    
    <li><a href="/research/" class="page-link">Research</a>
    <ul class="sub-menu">
    
    <li><a href="/research/projects/">Projects</a></li>
    
    <li><a href="/research/publications/">Publications</a></li>
    
    <li><a href="/research/pop/">Publish-or-Perish stats</a></li>
    
    </ul>
    
    </li>
    
    
    <li><a href="/teaching/" class="page-link">Teaching</a>
    <ul class="sub-menu">
    
    <li><a href="/teaching/resources/">Materials and Resources</a></li>
    
    </ul>
    
    </li>
    
    
    <li><a href="/about/" class="page-link">About</a>
    <ul class="sub-menu">
    
    <li><a href="/about/cv/">CV</a></li>
    
    </ul>
    
    </li>
    
    
     <li><a href="/blog/" class="page-link">Bits and Pieces</a>
    
    </li>
    
    </ul>


     </div>  
    </nav>

  </div>

</header>


    <div class="page-content">
      <div class="wrapper">
        <div class="post">

  <header class="post-header">
    <h1 class="post-title">Publications</h1>
  </header>

  <article class="post-content">
    <h1 id="bibliography">Bibliography</h1>

<table>
  <tbody>
    <tr>
      <td><a href="#book"><strong>Book</strong></a></td>
      <td><a href="#book-chapter"><strong>Book Chapter</strong></a></td>
      <td><a href="#journals"><strong>Journals</strong></a></td>
      <td><a href="#conferences-and-announcements"><strong>Conferences and Announcements</strong></a></td>
      <td><a href="#other"><strong>Other</strong></a></td>
    </tr>
  </tbody>
</table>

<h2 id="book">Book</h2>

<ol class="bibliography"><li><span id="OSTHbook">Bezjak, S., Clyburne-Sherin, A., Conzett, P., Fernandes, P. L., Görögh, E., Helbig, K., … Verbakel, E. (2018). <i>The Open Science Training Handbook</i>. https://doi.org/10.5281/zenodo.1212495</span></li></ol>

<h2 id="book-chapter">Book Chapter</h2>

<ol class="bibliography"><li><span id="DataMiningProteomicsGrid">Psomopoulos, F. E., &amp; Mitkas, P. A. (2009). Data Mining in Proteomics using Grid Computing. In M. Cannataro (Ed.), <i>Handbook of Research on Computational Grid Technologies for LifeSciences, Biomedicine and Healthcare</i> (pp. 245–267). UK: IGI Global. https://doi.org/10.4018/978-1-60566-374-6</span></li></ol>

<h2 id="journals">Journals</h2>

<h3 id="2019">2019</h3>

<ol class="bibliography"><li><span id="BISS_2019_ELIXIR">Kuzak, M., Harrow, J., Andrea Martinez, P., E Psomopoulos, F., &amp; Via, A. (2019). ELIXIR Europe on the Road to Sustainable Research Software. <i>Biodiversity Information Science and Standards</i>, <i>3</i>, e37677. https://doi.org/10.3897/biss.3.37677</span></li>
<li><span id="PARLAPANI2019">Parlapani, F. F., Michailidou, S., Anagnostopoulos, D. A., Koromilas, S., Kios, K., Pasentsis, K., … Boziaris, I. S. (2019). Bacterial communities and potential spoilage markers of whole blue crab (Callinectes sapidus) stored under commercial simulated conditions. <i>Food Microbiology</i>, <i>82</i>, 325–333. https://doi.org/10.1016/j.fm.2019.03.011</span></li>
<li><span id="Kintsakis2019">Kintsakis, A. M., Psomopoulos, F. E., &amp; Mitkas, P. A. (2019). Reinforcement Learning based scheduling in a workflow management system. <i>Engineering Applications of Artificial Intelligence</i>, <i>81</i>, 94–106. https://doi.org/10.1016/j.engappai.2019.02.013</span></li>
<li><span id="Agathangelidis2019">Agathangelidis, A., Psomopoulos, F., &amp; Stamatopoulos, K. (2019). Stereotyped B Cell Receptor Immunoglobulins in B Cell Lymphomas. <i>Methods in Molecular Biology: "Lymphoma: Methods and Protocols"</i>, 139–155. https://doi.org/10.1007/978-1-4939-9151-8_7</span></li>
<li><span id="DataJournalDDPIG">Wu, M., Psomopoulos, F., Khalsa, S. J., &amp; de Waard, A. (2019). Data Discovery Paradigms: User Requirements and Recommendations for Data Repositories. <i>Data Science Journal</i>, <i>18</i>(1), 13. https://doi.org/10.5334/dsj-2019-003</span></li></ol>

<h3 id="2018">2018</h3>

<ol class="bibliography"><li><span id="gavriilaki2018pre">Gavriilaki, E., Touloumenidou, T., Sakellari, I., Batsis, I., Mallouri, D., Psomopoulos, F., … Anagnostopoulos, A. (2018). Pre-Transplant Genetic Susceptibility in Adult Allogeneic Hematopoietic Cell Transplant Recipients: Incidence and Clinical Relevance in Transplant-Associated Thrombotic Microangiopathy. <i>Blood</i>, <i>132</i>(Suppl 1), 3401–3401. https://doi.org/10.1182/blood-2018-99-112041</span></li>
<li><span id="vardi2018longitudinal">Vardi, A., Vlachonikola, E., Ioannou, N., Psomopoulos, F., Kotta, K., Papazoglou, D., … Hadzidimitriou, A. (2018). Longitudinal High-Throughput T Cell Repertoire Profiling of Chronic Lymphocytic Leukemia Patients Under Different Types of Treatment: Implications for Combination Strategies. <i>Blood</i>, <i>132</i>(Suppl 1), 4400–4400. https://doi.org/10.1182/blood-2018-99-118006</span></li>
<li><span id="IJC2018TP63">Papakonstantinou, N., Ntoufa, S., Tsagiopoulou, M., Moysiadis, T., Bhoi, S., Malousi, A., … Stamatopoulos, K. (2018). Integrated epigenomic and transcriptomic analysis reveals TP63 as a novel player in clinically aggressive chronic lymphocytic leukemia. <i>International Journal of Cancer</i>, <i>0</i>(ja). https://doi.org/10.1002/ijc.31999</span></li>
<li><span id="FoodMicrobiology2018">Parlapani, F. F., Michailidou, S., Anagnostopoulos, D. A., Sakellariou, A. K., Pasentsis, K., Psomopoulos, F., … Boziaris, I. S. (2018). Microbial spoilage investigation of thawed common cuttlefish (Sepia officinalis) stored at 2 °C using next generation sequencing and volatilome analysis. <i>Food Microbiology</i>, <i>76</i>, 518–525. https://doi.org/10.1016/j.fm.2018.08.004</span></li>
<li><span id="ElevenTips">Gregory, K., Khalsa, S. J., Michener, W. K., Psomopoulos, F. E., de Waard, A., &amp; Wu, M. (2018). Eleven quick tips for finding research data. <i>PLOS Computational Biology</i>, <i>14</i>(4), 1–7. https://doi.org/10.1371/journal.pcbi.1006038</span></li></ol>

<h3 id="2017">2017</h3>

<ol class="bibliography"><li><span id="Hermes2017">Kintsakis, A. M., Psomopoulos, F. E., Symeonidis, A. L., &amp; Mitkas, P. A. (2017). Hermes: Seamless delivery of containerized bioinformatics workflows in hybrid cloud (HTC) environments. <i>SoftwareX</i>, <i>6</i>(Supplement C), 217–224. https://doi.org/10.1016/j.softx.2017.07.007</span></li>
<li><span id="SoilMetagenomics2017">Kalivas, A., Ganopoulos, I., Psomopoulos, F., Grigoriadis, I., Xanthopoulou, A., Hatzigiannakis, E., … Madesis, P. (2017). Comparative metagenomics reveals alterations in the soil bacterial community driven by N-fertilizer and Amino 16® application in lettuce. <i>Genomics Data</i>, <i>14</i>(Supplement C), 14–17. https://doi.org/10.1016/j.gdata.2017.07.013</span></li>
<li><span id="XANTHOPOULOU201750">Xanthopoulou, A., Ganopoulos, I., Psomopoulos, F., Manioudaki, M., Moysiadis, T., Kapazoglou, A., … Madesis, P. (2017). De novo comparative transcriptome analysis of genes involved in fruit morphology of pumpkin cultivars with extreme size difference and development of EST-SSR markers. <i>Gene</i>, <i>622</i>(Supplement C), 50–66. https://doi.org/10.1016/j.gene.2017.04.035</span></li>
<li><span id="RAPPRobotics2017">Zieliński, C., Stefańczyk, M., Kornuta, T., Figat, M., Dudek, W., Szynkiewicz, W., … Iturburu, M. (2017). Variable structure robot control systems: The RAPP approach. <i>Robotics and Autonomous Systems</i>, <i>94</i>(Supplement C), 226–244. https://doi.org/10.1016/j.robot.2017.05.002</span></li>
<li><span id="BioPaxViz2017">Psomopoulos, F. E., Vitsios, D. M., Baichoo, S., &amp; Ouzounis, C. A. (2017). BioPAXViz: a cytoscape application for the visual exploration of metabolic pathway evolution. <i>Bioinformatics</i>, <i>33</i>(9), 1418–1420. https://doi.org/10.1093/bioinformatics/btw813</span></li>
<li><span id="Tsardoulias2017">Tsardoulias, E. G., Kintsakis, A. M., Panayiotou, K., Thallas, A. G., Reppou, S. E., Karagiannis, G. T., … Mitkas, P. A. (2017). Towards an integrated robotics architecture for social inclusion – The RAPP paradigm. <i>Journal of Cognitive Systems Research</i>, <i>43</i>, 157–173. https://doi.org/10.1016/j.cogsys.2016.08.004</span></li></ol>

<h3 id="2016">2016</h3>

<ol class="bibliography"><li><span id="KintsakisBigData2016">Kintsakis, A. M., Psomopoulos, F. E., &amp; Mitkas, P. A. (2016). Data-aware optimization of bioinformatics workflows in hybrid clouds. <i>Journal of Big Data</i>, <i>3</i>(1), 20. https://doi.org/10.1186/s40537-016-0055-2</span></li>
<li><span id="PlantOmics2016">Zambounis, A., Psomopoulos, F. E., Ganopoulos, I., Avramidou, E., Aravanopoulos, F. A., Tsaftaris, A., &amp; Madesis, P. (2016). In silico analysis of the LRR receptor-like serine threonine kinases subfamily in Morus notabilis. <i>Plant Omics Journal</i>, <i>9</i>(5), 319–326. https://doi.org/10.21475/poj.09.05.16.pne126</span></li>
<li><span id="Reppou2016">Reppou, S. E., Tsardoulias, E. G., Kintsakis, A. M., Symeonidis, A. L., Mitkas, P. A., Psomopoulos, F. E., … Gkiokas, A. (2016). RAPP: A Robotic-Oriented Ecosystem for Delivering Smart User Empowering Applications for Older People. <i>International Journal of Social Robotics</i>, 1–14. https://doi.org/10.1007/s12369-016-0361-z</span></li>
<li><span id="Chatzidimopoulos2016">Chatzidimopoulos, M., Psomopoulos, F., Malandrakis, E. E., Ganopoulos, I., Madesis, P., Vellios, E. K., &amp; Drogoudi, P. (2016). Comparative Genomics of Botrytis cinerea Strains with Differential Multi-Drug Resistance. <i>Frontiers in Plant Science</i>, <i>7</i>(April), 1–3. https://doi.org/10.3389/fpls.2016.00554</span></li>
<li><span id="Xanthopoulou2016">Xanthopoulou, A., Psomopoulos, F., Ganopoulos, I., Manioudaki, M., Tsaftaris, A., Nianiou-Obeidat, I., &amp; Madesis, P. (2016). De novo transcriptome assembly of two contrasting pumpkin cultivars. <i>Genomics Data</i>, <i>7</i>, 200–201. https://doi.org/10.1016/j.gdata.2016.01.006</span></li></ol>

<h3 id="2015">2015</h3>

<ol class="bibliography"><li><span id="Duarte2015">Duarte, A. M. S., Psomopoulos, F. E., Blanchet, C., Bonvin, A. M. J. J., Corpas, M., Franc, A., … Suhr, S. B. (2015). Future opportunities and trends for e-infrastructures and life sciences: going beyond the grid to enable life science data analysis. <i>Frontiers in Genetics</i>, <i>6</i>(June), 197. https://doi.org/10.3389/fgene.2015.00197</span></li>
<li><span id="Vitsios2015">Vitsios, D. M., Psomopoulos, F. E., Mitkas, P. a., &amp; Ouzounis, C. a. (2015). Inference of Pathway Decomposition Across Multiple Species Through Gene Clustering. <i>International Journal on Artificial Intelligence Tools</i>, <i>24</i>(01), 1540003. https://doi.org/10.1142/S0218213015400035</span></li></ol>

<h3 id="2013">2013</h3>

<ol class="bibliography"><li><span id="FuzzyBMCBioinformatics">Psomopoulos, F. E., Mitkas, P. A., &amp; Ouzounis, C. A. (2013). Detection of genomic idiosyncrasies using fuzzy phylogenetic profiles. <i>PloS One</i>, <i>8</i>(1), e52854. https://doi.org/10.1371/journal.pone.0052854</span></li></ol>

<h3 id="2012">2012</h3>

<ol class="bibliography"><li><span id="GenesChlamydiae">Psomopoulos, F. E., Siarkou, V. I., Papanikolaou, N., Iliopoulos, I., Tsaftaris, A. S., Promponas, V. J., &amp; Ouzounis, C. A. (2012). The chlamydiales pangenome revisited: structural stability and functional coherence. <i>Genes</i>, <i>3</i>(2), 291–319. https://doi.org/10.3390/genes3020291</span></li>
<li><span id="MultiLevelClusteringJournal">Psomopoulos, F. E., &amp; Mitkas, P. A. (2012). Multi-Level Clustering of Phylogenetic Profiles. <i>International Journal on Artificial Intelligence Tools (Special Issue on Bioinformatics and Bioengineering)</i>, <i>21</i>(5), 1240023. https://doi.org/10.1142/S0218213012400234</span></li></ol>

<h3 id="2010">2010</h3>

<ol class="bibliography"><li><span id="Psomopoulos2010a">Psomopoulos, F. E., &amp; Mitkas, P. A. (2010). Bioinformatics algorithm development for Grid environments. <i>Journal of Systems and Software</i>, <i>83</i>(7), 1249–1257. https://doi.org/10.1016/j.jss.2010.01.051</span></li></ol>

<h3 id="2009">2009</h3>

<ol class="bibliography"><li><span id="gMolKnotMolSimul">Psomopoulos, F. E., Mitkas, P. A., Krinas, C. S., &amp; Demetropoulos, I. N. (2009). A grid-enabled algorithm yields figure-eight molecular knot. <i>Molecular Simulation</i>, <i>35</i>(9), 725–736. https://doi.org/10.1080/08927020902833103</span></li></ol>

<h2 id="conferences-and-announcements">Conferences and Announcements</h2>

<h3 id="2019-1">2019</h3>

<ol class="bibliography"><li><span id="ADBIS2019">Kotouza, M. T., Psomopoulos, F. E., &amp; Mitkas, P. A. (2019). A Dockerized String Analysis Workflow for Big Data. In <i>23rd European Conference on Advances in Databases and Information Systems, ASBIS 2019, Bled, Slovenia, September 8-11, 2019</i> (pp. 564–569). https://doi.org/10.1007/978-3-030-30278-8_55</span></li></ol>

<h3 id="2018-1">2018</h3>

<ol class="bibliography"><li><span id="BDCAT2018">Kotouza, M. T., Vavliakis, K. N., Psomopoulos, F. E., &amp; Mitkas, P. A. (2018). A hierarchical multi-metric framework for item clustering. In <i>5th IEEE/ACM International Conference on Big Data Computing Applications and Technologies, BDCAT 2018, Zurich, Switzerland, December 17-20, 2018</i> (pp. 191–197). IEEE. https://doi.org/10.1109/BDCAT.2018.00031</span></li>
<li><span id="WSSSPE61">Kuzak, M., Harrow, J., Jimenez, R. C., Martinez, P., Psomopoulos, F. E., Svobodová Vařeková, R., &amp; Via, A. (2018). Lesson development for Open Source Software best practices adoption. In <i>Working Towards Sustainable Software for Science Practice and Experiences</i>. Rhodes, Greece. Retrieved from https://doi.org/10.5281/zenodo.1420462</span></li>
<li><span id="14thAIAI">Tsarouchis, S.-F., Kotouza, M. T., Psomopoulos, F. E., &amp; Mitkas, P. A. (2018). A Multi-metric Algorithm for Hierarchical Clustering of Same-Length Protein Sequences. In <i>7th Mining Humanistic Data Workshop (MHDW2018) in conjunction with the 14th International Conference on Artificial Intelligence Applications and Innovations (AIAI 2018)</i> (pp. 189–199). Rhodes, Greece. https://doi.org/10.1007/978-3-319-92016-0_18</span></li></ol>

<h3 id="2017-1">2017</h3>

<ol class="bibliography"><li><span id="ECCB2017Pangenomes">Mpatziakas, A., Psomopoulos, F. E., Moysiadis, T., &amp; Sgardelis, S. (2017). Computing pangenome statistics in R, [v1; not peer reviewed]. In <i>F1000research 2017 (5th Annual International Conference on Intelligent Systems for Molecular Biology (ISMB,) / 16th European Conference on Computational Biology (ECCB))</i> (Vol. 6(ISCB Comm J), p. 1). https://doi.org/10.7490/f1000research.1114765.1</span></li>
<li><span id="DI4R2017">Culina, A., Kuzak, M., &amp; Psomopoulos, F. E. (2017). Training Life Scientists in Open Reproducible Research; a Data Science scenario. In <i>Digital Infrastructures for Research 2017</i> (pp. 1–2). Brussels, Belgium. Retrieved from https://doi.org/10.6084/m9.figshare.5644279.v1</span></li>
<li><span id="gemenetzi2017high">Gemenetzi, K., Stalika, E., Vardi, A., Psomopoulos, F. E., Minga, E., Anagnostopoulos, A., … Sutton, L. A. (2017). HIGH THROUGHPUT IMMUNOPROFILING OF CHRONIC LYMPHOCYTIC LEUKEMIA PATIENTS ASSIGNED TO STEREOTYPED SUBSET# 4: NOVEL INSIGHTS INTO THE DEPTH, DIVERSITY AND TEMPORAL DYNAMICS OF CLONAL EVOLUTION. In <i>HAEMATOLOGICA</i> (Vol. 102, pp. 67–67). FERRATA STORTI FOUNDATION VIA GIUSEPPE BELLI 4, 27100 PAVIA, ITALY.</span></li>
<li><span id="BloodASH_2017b">Agathangelidis, A., Galigalidou, C., Scarfo, L., Rovida, A., Gounari, M., Psomopoulos, F., … Ghia, P. (2017). CLL-Specific B Cell Receptor Immunoglobulin Stereotypes Are Very Infrequent in Low Count Monoclonal B Cell Lymphocytosis: Implications for Disease Ontogeny. In <i>Blood</i> (Vol. 130, pp. 1710–1710). American Society of Hematology. Retrieved from http://www.bloodjournal.org/content/130/Suppl_1/1710</span></li>
<li><span id="BloodASH_2017a">Agathangelidis, A., Galigalidou, C., Scarfo, L., Rovida, A., Moysiadis, T., Vlachonikola, E., … Ghia, P. (2017). High Throughput Immunoprofiling of Low Count B Lymphocytosis Reveals a Distinct T Cell Receptor Repertoire from Chronic Lymphocytic Leukemia. In <i>Blood</i> (Vol. 130, pp. 4296–4296). American Society of Hematology. Retrieved from http://www.bloodjournal.org/content/130/Suppl_1/4296</span></li></ol>

<h3 id="2016-1">2016</h3>

<ol class="bibliography"><li><span id="HSBMB2016">Mokos, P., Psomopoulos, F. E., Ainali, C., Argiriou, A., Charalampidou, A., Hadzopoulou-Cladaras, M., &amp; Dafou, D. (2016). Network analysis and regularized regression modeling identifies key regulatory molecular events underlying hepatocellular carcinoma dedifferentiation. In <i> 67th Annual Conference of the Hellenic Society for Biochemistry and Molecular Biology (HSBMB)</i> (pp. 1–4). “Karolos Papoulias” Conference Center, University of Ioannina, Ioannina, Greece.</span></li>
<li><span id="HeCrA16">Mokos, P., Psomopoulos, F. E., Ainali, C., Argiriou, A., Charalampidou, A., Hadzopoulou-Cladaras, M., &amp; Dafou, D. (2016). Integrated network analysis and regularized regression modeling reveal regulatory miRNA-mRNA networks associated with hepatocellular carcinoma dedifferentiation. In <i>8th Conference of the Hellenic Crystallographic Association (HeCrA) / 11th Conference of the Hellenic Society for Computational Biology and Bioinformatics (HeCrA-HSCBB16)</i> (pp. 1–2). Agricultural University of Athens, Greece.</span></li>
<li><span id="PlantBreedingXanthopoulou">Xanthopoulou, A., Psomopoulos, F., Ganopoulos, I., Kalyvas, A., Tsaftaris, A., Madesis, P., &amp; Nianiou-Obeidat, I. (2016). De novo sequencing and comparative analysis of the transcriptome of two Cucurbita pepo cultivars (in Greek). In <i>16th Conference of the Hellenic Plant Breeding Society</i> (pp. 1–2). Florina, Greece.</span></li>
<li><span id="DI4R2016">Via, A., Fernandes, P., Korpelainen, E., Duarte, A. M. S., &amp; Psomopoulos, F. E. (2016). E-Infrastructures for the Bioinformatics long tail of science: a user perspective. In <i>Digital Infrastructures for Research 2016</i> (pp. 1–2). Krakow, Poland.</span></li>
<li><span id="8thRDA">Psomopoulos, F. E. (2016). Managing the Data Ecosystem in Life Sciences through Bioinformatics. In <i>8th RDA Plenary</i> (pp. 1–1). Denver, Colorado, USA.</span></li>
<li><span id="12thAIAI">Stergiadis, E., Kintsakis, A. M., Psomopoulos, F. E., &amp; Mitkas, P. A. (2016). A scalable Grid Computing framework for extensible phylogenetic profile construction. In <i>5th Mining Humanistic Data Workshop (MHDW2016) in conjunction with the 12th International Conference on Artificial Intelligence Applications and Innovations (AIAI 2016)</i> (pp. 1–8). Thessaloniki, Greece.</span></li>
<li><span id="ECCB2016Pangenomes">Psomopoulos, F. E., Kintsakis, A. M., &amp; Mitkas, P. A. (2016). A pan-genome approach and application to species with photosynthetic capabilities, [v1; not peer reviewed]. In <i>F1000Research 2016</i> (Vol. 5(ECCB 2016, p. 1). https://doi.org/10.7490/f1000research.1112964.1</span></li>
<li><span id="ECCB2016FedCloud">Psomopoulos, F. E., Korpelainen, E., Mattila, K., &amp; Scardaci, D. (2016). Bioinformatics resources on EGI Federated Cloud, [v1; not peer reviewed]. In <i>F1000Research 2016</i> (Vol. 5(ECCB 2016, p. 1). https://doi.org/10.7490/f1000research.1112963.1</span></li></ol>

<h3 id="2015-1">2015</h3>

<ol class="bibliography"><li><span id="26thPHCErrorCorrection">Kintsakis, A., Vrousgou, O., Vardi, A., Karypidou, M., Stalika, E., Korabou, E., … Psomopoulos, F. E. (2015). Large-scale detection of sequencing errors using immunogenetic data of β chain T cell receptors (in Greek). In <i>26th Panhellenic Haematology Congress</i> (pp. 1–2). Athens, Greece.</span></li>
<li><span id="26thPHCAward">Ntoufa, S., Papakonstantinou, N., Tsagiopoulou, M., Moisiadis, T., Malousi, A., Papazoglou, D., … Stamatopoulos, K. (2015). Distinct methylation patterns in subgroups of patients with Chronic Lymphocytic Leukemia reveal TP63 as a new player in clinical aggressiveness ("Arkagathos Gouttas" Prize award). In <i>26th Panhellenic Haematology Congress</i> (pp. 1–2). Athens, Greece.</span></li>
<li><span id="EEBMB2015">Mokos, P., Psomopoulos, F. E., Ainali, C., Charalampidou, A., Hadzopoulou-Cladaras, M., &amp; Dafou, D. (2015). Large integrative analysis for the identification of novel molecular targets in hepatocellular carcinoma. In <i>66th Congress of the Hellenic Society of Biochemistry and Molecular Biology</i> (pp. 1–4). Eugenides Foundation, Athens, Greece.</span></li>
<li><span id="Duarte2015a">Duarte, A., Psomopoulos, F. E., Sipos, G., Ferreira, N., &amp; Scardaci, D. (2015). EGI Support for Genome Analysis and Protein Folding - A Virtual Team for Life Sciences. In <i>International Symposium on Grids and Clouds ( ISGC ) 2015</i> (p. 2).</span></li>
<li><span id="Psomopoulos2015">Psomopoulos, F. E., Vrousgou, O. T., &amp; Mitkas, P. A. (2015). Large-scale modular comparative genomics: the Grid approach [v1; not peer reviewed]. In <i>F1000Research 2015</i> (Vol. 4(ISCB Com, p. 1). https://doi.org/10.7490/f1000research.1110127.1</span></li>
<li><span id="Vrousgou2015">Vrousgou, O. T., Psomopoulos, F. E., &amp; Mitkas, P. A. (2015). A grid-enabled modular framework for efficient sequence analysis workflows. In <i>16th International Conference on Engineering Applications of Neural Networks</i> (p. 10). Island of Rhodes.</span></li></ol>

<h3 id="2014">2014</h3>

<ol class="bibliography"><li><span id="Psomopoulos2014">Psomopoulos, F. E., &amp; Mitkas, P. A. (2014). Algebraic Interpretations Towards Clustering Protein Homology Data. In <i>Artificial Intelligence Applications and Innovations, IFIP Advances in Information and Communication Technology Volume 437</i>. https://doi.org/10.1007/978-3-662-44722-2_15</span></li>
<li><span id="Psomopoulos2014a">Psomopoulos, F. E., &amp; Ouzounis, C. A. (2014). Computation and visualization of ancestral pathway reconstruction and inference. In <i>ECCB 2014</i> (Vol. 8, p. 52854).</span></li>
<li><span id="Psomopoulos2014b">Psomopoulos, F., Tsardoulias, E., Giokas, A., Zielinski, C., Prunet, V., Trochidis, I., … Mitkas, P. A. (2014). RAPP System Architecture. In <i>Workshop on Assistance and Service Robotics in a Human Environment (ASROB), in conjunction with the IEEE/RSJ International Conference on Intelligent Robots and Systems (IROS’14)</i> (p. 8). Chicago, Illinois, USA.</span></li></ol>

<h3 id="2012-1">2012</h3>

<ol class="bibliography"><li><span id="HSCBB12Gluco">Psomopoulos, F. E., Argiriou, A., Tsaftaris, A. S., &amp; Ouzounis, C. A. (2012). A Pangenome Analysis of the Glucosinolate Pathway in Plants. In <i>7th conference of the Hellenic Society for Computational Biology &amp; Bioinformatics - HSCBB ’12</i> (p. 63). FORTH, Heraklion, Crete, Greece: Hellenic Society for Computational Biology &amp; Bioinformatics.</span></li>
<li><span id="HSCBB12Ancestral">Psomopoulos, F. E., &amp; Ouzounis, C. A. (2012). Ancestral Reconstruction of Metabolic Pathway Content at the Paleome Level. In <i>7th conference of the Hellenic Society for Computational Biology &amp; Bioinformatics - HSCBB ’12</i> (p. 90). FORTH, Heraklion, Crete, Greece: Hellenic Society for Computational Biology &amp; Bioinformatics.</span></li>
<li><span id="SFHMMY5">Vitsios, D., Psomopoulos, F. E., Mitkas, P. A., &amp; Ouzounis, C. A. (2012). Detection of closely correlated genes across genomes using pathway data. In <i>5th National Student Conference of Electrical and Computer Engineering</i> (pp. 111–117). Democritus University of Thrace, Xanthi, Greece.</span></li>
<li><span id="HSCBB12BioPax">Vitsios, D., Psomopoulos, F. E., &amp; Ouzounis, C. A. (2012). Null Entries for BioPAX Pathways, Visualization and Pathway Phylogenetic Profiling Based on Paxtools. In <i>7th conference of the Hellenic Society for Computational Biology &amp; Bioinformatics - HSCBB ’12</i> (p. 92). FORTH, Heraklion, Crete, Greece: Hellenic Society for Computational Biology &amp; Bioinformatics.</span></li></ol>

<h3 id="2011">2011</h3>

<ol class="bibliography"><li><span id="HSCBB11">Vitsios, D., Psomopoulos, F. E., Mitkas, P. A., &amp; Ouzounis, C. A. (2011). Detecting species evolution through metabolic pathways. In <i>6th Conference of the Hellenic Society For Computational Biology and Bioinformatics - HSCBB ’11</i> (p. 16). University of Patras Conference Center.</span></li></ol>

<h3 id="2010-1">2010</h3>

<ol class="bibliography"><li><span id="KyrchaPsomopoulosEGEEForum">Chatzidimitriou, K. C., Psomopoulos, F. E., &amp; Mitkas, P. A. (2010). Grid-enabled parameter initialization for high performance machine learning tasks. In <i>5th EGEE User Forum</i> (pp. 113–114). Uppsala, Sweden.</span></li>
<li><span id="Psomopoulos2010">Psomopoulos, F. E., &amp; Mitkas, P. A. (2010). Multi Level Clustering of Phylogenetic Profiles. In <i>10th IEEE International Conference on Bioinformatics and Bioingineering</i> (pp. 308–309). Philadelphia, PA, USA: IEEE. https://doi.org/10.1109/BIBE.2010.67</span></li>
<li><span id="HSCBB10">Psomopoulos, F. E., Mitkas, P. A., &amp; Ouzounis, C. A. (2010). Clustering of discrete and fuzzy phylogenetic profiles. In <i>5th Conference of the Hellenic Society For Computational Biology and Bioinformatics - HSCBB ’10</i> (p. 58). Alexandroupoli, Greece.</span></li>
<li><span id="TzimaPsomopoulosEGEEForum">Tzima, F. A., Psomopoulos, F. E., &amp; Mitkas, P. A. (2010). An investigation of the effect of clustering-based initialization on Learning Classifiers Systems’ effectiveness: leveraging the Grid infrastructure. In <i>5th EGEE User Forum</i> (pp. 111–112). Uppsala, Sweden.</span></li></ol>

<h3 id="2009-1">2009</h3>

<ol class="bibliography"><li><span id="HSCBB09">Karagiannis, K. M., Psomopoulos, F. E., &amp; Mitkas, P. A. (2009). Multi Level Clustering of Phylogenetic Profiles. In <i>4th Conference of the Hellenic Society For Computational Biology and Bioinformatics - HSCBB ’09</i> (p. 1). Athens, Greece: Hellenic Society for Computational Biology &amp; Bioinformatics.</span></li>
<li><span id="FpsomPCI2009">Psomopoulos, F. E., &amp; Mitkas, P. A. (2009). BADGE: Bioinformatics Algorithm Development for Grid Environments. In <i>13th Panhellenic Conference on Informatics</i> (pp. 93–107). Corfu, Greece.</span></li>
<li><span id="ASSISTForum2009">Mitkas, P. A., Delopoulos, A., Vavliakis, K., Maramis, C., Falelakis, M., Psomopoulos, F. E., … Agorastos, T. (2009). Combining multiple sources towards cervical cancer prognosis – the ASSIST platform (in Greek). In <i>8th Scientific Forum on "New trends in prognosis and treatment of cervical cancer"</i> (pp. 109–118). Thessaloniki, Greece.</span></li></ol>

<h3 id="2008">2008</h3>

<ol class="bibliography"><li><span id="WCCAssist">Agorastos, T., Mitkas, P. A., Falelakis, M., Psomopoulos, F. E., Delopoulos, A. N., Symeonidis, A. L., … Maglaveras, N. (2008). Large Scale Association Studies Using Unified Data for Cervical Cancer and beyond: The ASSIST Project. In <i>World Cancer Congress</i> (p. 1). Geneva, Switzerland.</span></li>
<li><span id="GkekasPsomopoulosSFHMMY">Gkekas, C. N., Psomopoulos, F. E., &amp; Mitkas, P. A. (2008). A Parallel Data Mining Methodology for Protein Function Prediction Utilizing Finite State Automata. In <i>2nd Electrical and Computer Engineering Student Conference</i> (p. 6). Athens, Greece.</span></li>
<li><span id="CkekasFpsomSamos">Gkekas, C. N., Psomopoulos, F. E., &amp; Mitkas, P. A. (2008). Exploiting parallel data mining processing for protein annotation. In <i>Student EUREKA 2008: 2nd Panhellenic Scientific Student Conference</i> (pp. 242–252). Samos, Greece.</span></li>
<li><span id="GkekasPsomopoulosEGEEForum">Gkekas, C. N., Psomopoulos, F. E., &amp; Mitkas, P. A. (2008). A Parallel Data Mining Application for Gene Ontology Term Prediction. In <i>3rd EGEE User Forum</i> (p. 1). Clermont-Ferrand, France.</span></li>
<li><span id="EsbmeAssist">Mitkas, P. A., Maramis, C., Delopoulos, A. N., Symeonidis, A. L., Diplaris, S., Falelakis, M., … Tatsis, A. (2008). ASSIST: Employing Inference and Semantic Technologies to Facilitate Association Studies on Cervical Cancer. In <i>6th European Symposium on Biomedical Engineering</i>. Chania, Greece.</span></li>
<li><span id="MprouzaFpsomSamos">Mprouza, I. K., Psomopoulos, F. E., &amp; Mitkas, P. A. (2008). AMoS: Agent-based Molecular Simulations. In <i>Student EUREKA 2008: 2nd Panhellenic Scientific Student Conference</i> (pp. 175–186). Samos, Greece.</span></li>
<li><span id="HSCBB08">Psomopoulos, F. E., &amp; Mitkas, P. A. (2008). Sizing Up: Bioinformatics in a Grid Context. In <i>3rd Conference of the Hellenic Society For Computational Biology and Bioinformatics - HSCBB ’08</i> (p. 1). Thessaloniki, Greece: Hellenic Society for Computational Biology &amp; Bioinformatics.</span></li>
<li><span id="PsomopoulosDemetropoulosHellasGrid">Psomopoulos, F. E., Mitkas, P. A., Krinas, C. S., &amp; Demetropoulos, I. N. (2008). G-MolKnot: A grid enabled systematic algorithm to produce open molecular knots. In <i>1st HellasGrid User Forum</i> (pp. 24–25). Athens, Greece.</span></li></ol>

<h3 id="2007">2007</h3>

<ol class="bibliography"><li><span id="GkekasPsomopoulosBioacademy">Gkekas, C. N., Psomopoulos, F. E., &amp; Mitkas, P. A. (2007). Modeling Gene Ontology Terms using Finite State Automata. In <i>Hellenic Bioinformatics and Medical Informatics Meeting</i> (p. 1). Biomedical Research Foundation, Academy of Athens, Greece.</span></li>
<li><span id="ASSISTBioacademy">Mitkas, P. A., Delopoulos, A. N., Symeonidis, A. L., &amp; Psomopoulos, F. E. (2007). A Framework for Semantic Data Integration and Inferencing on Cervical Cancer. In <i>Hellenic Bioinformatics and Medical Informatics Meeting</i> (p. 1). Biomedical Research Foundation, Academy of Athens, Greece.</span></li>
<li><span id="MprouzaPsomopoulosBioacademy">Mprouza, I. K., Psomopoulos, F. E., &amp; Mitkas, P. A. (2007). Simulating molecular dynamics through intelligent software agents. In <i>Hellenic Bioinformatics and Medical Informatics Meeting</i> (p. 1). Biomedical Research Foundation, Academy of Athens, Greece.</span></li></ol>

<h3 id="2006">2006</h3>

<ol class="bibliography"><li><span id="PolychroniadouPsomopoulosGClass">Polychroniadou, H. E., Psomopoulos, F. E., &amp; Mitkas, P. A. (2006). G-Class: A Divide and Conquer Application for Grid Protein Classification. In <i>Proceedings of the 2nd ADMKD 2006: Workshop on Data Mining and Knowledge Discovery (in conjunction with ADBIS 2006: The 10th East-European Conference on Advances in Databases and Information Systems)</i> (pp. 121–132). Thessaloniki, Greece: Springer-Verlag.</span></li>
<li><span id="PsomopoulosHDMS">Psomopoulos, F. E., &amp; Mitkas, P. A. (2006). PROTEAS: A Finite State Automata based data mining algorithm for rule extraction in protein classification. In <i>Proceedings of the 5th Hellenic Data Management Symposium</i> (pp. 118–126). Thessaloniki, Greece.</span></li></ol>

<h3 id="2005">2005</h3>

<ol class="bibliography"><li><span id="LoutrakiPsomopoulos">Psomopoulos, F. E., &amp; Mitkas, P. A. (2005). A protein classification engine based on stochastic finite state automata. In <i>Lecture Series on Computer and Computational Sciences VSP/Brill (Proceedings of the Symposium 35: Computational Methods in Molecular Biology in conjunction with ICCMSE)</i> (Vol. 4B, pp. 1371–1374). Loutraki, Greece: Springer-Verlag.</span></li></ol>

<h3 id="2004">2004</h3>

<ol class="bibliography"><li><span id="PsomopoulosDiplarisPisa">Psomopoulos, F. E., Diplaris, S., &amp; Mitkas, P. A. (2004). A finite state automata based technique for protein classification rules induction. In <i>Proceedings of the Second European Workshop on Data Mining and Text Mining in Bioinformatics (in conjunction with ECML/PKDD)</i> (pp. 54–60). Pisa, Italy.</span></li></ol>

<h2 id="other">Other</h2>

<h3 id="short-articles">Short articles</h3>

<ol class="bibliography"><li><span id="erin_alison_becker_2019_3260609">Becker, E. A., Reiter, T., Psomopoulos, F., McKay, S. J., Mizzi, J. E., Williams, J., … Kretzschmar, W. (2019). datacarpentry/wrangling-genomics: Data Carpentry:
                   Genomics data wrangling and processing, June 2019. https://doi.org/10.5281/zenodo.3260609</span></li>
<li><span id="wilson_greg_2017_1064254">Wilson, G., Becker, E., McKay, S., Michonneau, F., Williams, J. J., Mayes, A. C., … gtehennepe. (2017). Data Carpentry Wrangling Genomics Lesson. https://doi.org/10.5281/zenodo.1064254</span></li>
<li><span id="Psomopoulos2016-RDAb">Psomopoulos, F. E. (2016). International Data Forum Opening Plenary Session: Maintaining Scientific Rigor and Enhancing Discovery. <i>RDA Newsletter</i>. Retrieved from https://www.rd-alliance.org/blogs/international-data-forum-opening-plenary-session-maintaining-scientific-rigor-and-enhancing</span></li>
<li><span id="Psomopoulos2016-RDAa">Psomopoulos, F. E. (2016). International Data Forum: Data Stories Session. <i>RDA Newsletter</i>. Retrieved from https://www.rd-alliance.org/blogs/international-data-forum-data-stories-session.html</span></li>
<li><span id="Psomopoulos2016-CSC">Psomopoulos, F. E. (2016). Training life scientists to analyse genomic data – can cloud make life easier? (interview). <i>@CSC Online Magazine Tells More</i>. Retrieved from https://www.csc.fi/web/atcsc/-/training-life-scientists-to-analyse-genomic-data-can-cloud-make-life-easier</span></li>
<li><span id="Psomopoulos2015a">Psomopoulos, F., &amp; Jimenez, R. (2015). EGI-ELIXIR project: Integrating datasets for bioinformatics. <i>EGI Inspire Newsletter #18</i>.</span></li>
<li><span id="Duarte2014">Duarte, A., &amp; Psomopoulos, F. (2014). Future opportunities and trends for e-infrastructures and life sciences. <i>EGI Inspire Newsletter #14</i>.</span></li>
<li><span id="Psomopoulos2008">Psomopoulos, F. E., Gkekas, C., &amp; Mitkas, P. A. (2008). Data Mining in Bioinformatics using Grid Computing (in Greek). <i>AUTH Grid Team, Aristotle University of Thessaloniki</i>.</span></li></ol>

<h3 id="thesis">Thesis</h3>

<ol class="bibliography"><li><span id="FpsomPhDThesis">Psomopoulos, F. E. (2010). <i>Parallel Data Mining and Analysis Algorithms in a Grid environment and applications in Bioinformatics</i> (PhD Thesis). Aristotle University of Thessaloniki, Greece.</span></li>
<li><span id="FpsomMasterThesis">Psomopoulos, F. E. (2004). <i>A finite state automata algorithm for the extraction of association rules and applications in protein data classification (Greek text only)</i> (Master Thesis). Aristotle University of Thessaloniki, Greece.</span></li></ol>


  </article>

</div>

      </div>
    </div>

    <footer class="site-footer">

  <div class="wrapper">

<!--     <h2 class="footer-heading">Cloud Computing and Bioinformatics</h2> -->

    <div class="footer-col-wrapper">
      <div class="footer-col  footer-col-1">
        <ul class="contact-list">
          <li><strong>Cloud Computing and Bioinformatics</strong></li>
          <li>Research Assistant Professor</li>
          <li><a href="mailto:fpsom-[at]-certh-dot-gr">fpsom-[at]-certh-dot-gr</a></li>
        </ul>
      </div>

      <div class="footer-col  footer-col-2">
        <ul class="social-media-list">
          
          <li>
            <a href="https://github.com/fpsom">
              <span class="icon  icon--github">
                <svg viewBox="0 0 16 16">
                  <path fill="#828282" d="M7.999,0.431c-4.285,0-7.76,3.474-7.76,7.761 c0,3.428,2.223,6.337,5.307,7.363c0.388,0.071,0.53-0.168,0.53-0.374c0-0.184-0.007-0.672-0.01-1.32 c-2.159,0.469-2.614-1.04-2.614-1.04c-0.353-0.896-0.862-1.135-0.862-1.135c-0.705-0.481,0.053-0.472,0.053-0.472 c0.779,0.055,1.189,0.8,1.189,0.8c0.692,1.186,1.816,0.843,2.258,0.645c0.071-0.502,0.271-0.843,0.493-1.037 C4.86,11.425,3.049,10.76,3.049,7.786c0-0.847,0.302-1.54,0.799-2.082C3.768,5.507,3.501,4.718,3.924,3.65 c0,0,0.652-0.209,2.134,0.796C6.677,4.273,7.34,4.187,8,4.184c0.659,0.003,1.323,0.089,1.943,0.261 c1.482-1.004,2.132-0.796,2.132-0.796c0.423,1.068,0.157,1.857,0.077,2.054c0.497,0.542,0.798,1.235,0.798,2.082 c0,2.981-1.814,3.637-3.543,3.829c0.279,0.24,0.527,0.713,0.527,1.437c0,1.037-0.01,1.874-0.01,2.129 c0,0.208,0.14,0.449,0.534,0.373c3.081-1.028,5.302-3.935,5.302-7.362C15.76,3.906,12.285,0.431,7.999,0.431z"/>
                </svg>
              </span>
              <span class="username">fpsom</span>
            </a>
          </li>
          

          
          <li>
            <a href="https://twitter.com/fopsom">
              <span class="icon  icon--twitter">
                <svg viewBox="0 0 16 16">
                  <path fill="#828282" d="M15.969,3.058c-0.586,0.26-1.217,0.436-1.878,0.515c0.675-0.405,1.194-1.045,1.438-1.809
                  c-0.632,0.375-1.332,0.647-2.076,0.793c-0.596-0.636-1.446-1.033-2.387-1.033c-1.806,0-3.27,1.464-3.27,3.27 c0,0.256,0.029,0.506,0.085,0.745C5.163,5.404,2.753,4.102,1.14,2.124C0.859,2.607,0.698,3.168,0.698,3.767 c0,1.134,0.577,2.135,1.455,2.722C1.616,6.472,1.112,6.325,0.671,6.08c0,0.014,0,0.027,0,0.041c0,1.584,1.127,2.906,2.623,3.206 C3.02,9.402,2.731,9.442,2.433,9.442c-0.211,0-0.416-0.021-0.615-0.059c0.416,1.299,1.624,2.245,3.055,2.271 c-1.119,0.877-2.529,1.4-4.061,1.4c-0.264,0-0.524-0.015-0.78-0.046c1.447,0.928,3.166,1.469,5.013,1.469 c6.015,0,9.304-4.983,9.304-9.304c0-0.142-0.003-0.283-0.009-0.423C14.976,4.29,15.531,3.714,15.969,3.058z"/>
                </svg>
              </span>

              <span class="username">fopsom</span>
            </a>
          </li>
          

        </ul>
      </div>

      <div class="footer-col  footer-col-3">
         <p class="text">
           Institute of Applied Biosciences (INAB), Center for Research and Technology Hellas (CERTH), Thermi 57001, Greece

      </div>
    </div>

  </div>

</footer>

    <!-- Global site tag (gtag.js) - Google Analytics -->
<script async src="https://www.googletagmanager.com/gtag/js?id=UA-17296390-9"></script>
<script>
  window.dataLayer = window.dataLayer || [];
  function gtag(){dataLayer.push(arguments);}
  gtag('js', new Date());

  gtag('config', 'UA-17296390-9');
</script>

  </body>

</html>
